knotx-stack-manager/src/main/packaging/bin/knotx

```diff
@@ -32,6 +32,9 @@ JMX_OPTS=""
 # You can specify hazelcast cluster options here. See http://docs.hazelcast.org/docs/3.6/manual/html-single/index.html#system-properties for the available options.
 HAZELCAST_OPTS="-Dhazelcast.max.no.heartbeat.seconds=5"
 
+# You can enable Vert.x metrics by uncommenting this line.
+# METRICS_OPTS="-Dvertx.metrics.options.enabled=true -Dvertx.metrics.options.registryName=knotx-dw"
```

Comment:
Can we please change this `knotx-dw` name to something more meaningful like: `knotx-metrics-registry`?